DigitalLearningSolutions.Data/Services/UserVerificationService.cs

```diff
@@ -83,6 +83,8 @@ string password
             var adminUserAssociatedWithDelegates = userDataService.GetAdminUserByEmailAddress(delegateEmail);
 
             var isSuitableAdmin = adminUserAssociatedWithDelegates != null &&
+                                  adminUserAssociatedWithDelegates.Active &&
+                                  adminUserAssociatedWithDelegates.Approved &&

```

Comment:
Could we just tweak the names of this method and the delegate equivalent?
`Verified` says "username and password match" to me, but doesn't imply active / approved.
I think adding an adjective to the names of the methods would do the trick. Something that implies both `active` and `approved`. "Active" would be ideal if it weren't a meaningful name already. "InUse"?

Reply:
Rather than trying to think of some other word to use, I've just gone with the simple approach of adding ActiveApproved